openBCIBoard.js

```diff
@@ -110,6 +111,8 @@ function OpenBCIFactory () {
   *
   *     - `verbose` {Boolean} - Print out useful debugging events
   *
+  *     - `debug` {Boolean} - Print out a raw dump of bytes sent and received
```

Comment:
Please add `(Default `false`)` additionally, I now see that `verbose` is missing the `(Default `false`)`, please add there too.